py4web/utils/grid.py

```diff
@@ -905,10 +905,18 @@ def render_table_body(self):
                 row_id = row["id"]
                 self.use_tablename = False
 
+            key = "%s.%s" % (self.tablename, '__row')
+            if self.formatters.get(key):
+                extra_class = self.formatters.get(key)(row)['_class']
+                extra_style = self.formatters.get(key)(row)['_style']
+            else:
+                extra_class= ''
+                extra_style= '
```

Comment:
This looks like it is missing it's second ' in `extra_style= '` vs `extra_style= ''`